app/controllers/media/objects_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Media::ObjectsController < ApplicationController
+  before_action :set_media_object, only: [:show, :edit, :update, :destroy]
+  skip_authorization_check
+
+  # GET /media/objects/1
+  # GET /media/objects/1.json
+  def show
+    style = params[:style]
+    send_file @media_object.item.path(style), type: @media_object.item_content_type, disposition: 'inline'
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
```

Comment:
Just from the rails resource generator, the controller and associated route were necessary for testing paperclip's overrides because of the non-public file store. I can remove it, but I wouldn't have been able to figure out if the paperclip configuration was right if I hadn't built it first. Will remove the comments though.